test/integration/utils.js

```diff
@@ -0,0 +1,53 @@
+import path from 'path';
+
+import { BrowserWindow } from 'electron';
+
+const UI_DIR = path.join(__dirname, '..', '..', 'lib', 'ui');
+const BROWSER_CHROME_URL = fileUrl(path.join(UI_DIR, 'browser-modern', 'browser.html'));
+
+function fileUrl(str) {
+  let pathName = path.resolve(str).replace(/\\/g, '/');
```

Comment:
That's a good point. I had similar problems when trying to make `npm run serve` great again. The LIBDIR wasn't declared when importing from the app folder, since that global was only added in webpacked files.